charts/tfy-agent/templates/tfy-agent-proxy-clusterrole-ns.yaml

```diff
@@ -24,7 +24,7 @@ rules:
 
   - apiGroups: [""]
     resources: ["pods"]
-    verbs: ["delete"]
+    verbs: ["delete", "get"]
```

Comment:
Lets try to remove the need for get pods

Reply:
This is not possible: https://github.com/truefoundry/tfy-k8s-controller/pull/241#discussion_r1848282554